codejail_service/apps/api/v0/views.py

```diff
@@ -92,21 +93,26 @@ def code_exec(request):
     Other responses are errors, with a JSON body containing further details.
     """
     if not CODEJAIL_ENABLED.is_enabled():
+        set_custom_attribute('codejail.exec', 'disabled.feature')
```

Comment:
- How about `codejail.exec.status` to better differentiate from the other attributes?
- Please follow https://github.com/openedx/edx-django-utils/issues/328 to annotate each of the attributes?

Reply:
Sure, I can add `.status`. Or maybe `.outcome` to avoid confusion with HTTP status codes.

I forgot about those annotations! I'll add those.

Reply:
I prefer `status` to `outcome`. I can't think of a better term, but I'm open. I also think that `http.status` would be an http status, and `codejail.exec.status`, would be a codejail exec status. :)

Reply:
OK, let's go with status. It bugs me a bit that the term generically means the current running state of something, but it gets used in HTTP (and other places) to mean the finished outcome or result, but... I'm not going to fight that convention very hard. :-)

Reply:
Understood. If you can get everyone to switch to `http.outcome`, I'd be content with that change here as well. 😜